cpp_modules/src/sc2_state/Sc2Constants.h

```diff
@@ -8,4 +8,6 @@ static constexpr int WORKER_PER_GEYSER = 3;
 static constexpr int WORKER_PER_MINERAL_FIELD = 2;
 static constexpr int MINERAL_PER_WORKER = 1;
 static constexpr int VESPENE_PER_WORKER = 1;
+static constexpr int TANK_SUPPLY = 3;
```

Comment:
Is the tanksupply and vikingsupply just to make sure we only have that many of them?

Reply:
This is multiplied with the number of tanks, as each tank takes up 3 supply